src/installermanual.cpp

```diff
@@ -50,7 +53,7 @@ QString InstallerManual::name() const
 
 QString InstallerManual::author() const
 {
-  return "Tannin";
+  return "Tannin, Holt59";

```

Comment:
You know this means future maintainer XD

Reply:
I even forgot I put that here. 🙃